mmv1/templates/terraform/update_encoder/spanner_database.go.erb

```diff
@@ -39,4 +39,5 @@ delete(obj, "name")
 delete(obj, "versionRetentionPeriod")
 delete(obj, "instance")
 delete(obj, "extraStatements")
+delete(obj, "enableDropProtection")
```

Comment:
```suggestion
```
For update_decoder, let add a condition check for ddl specific decoder -- `if obj["versionRetentionPeriod"] != nil || obj["extraStatements"] != nil`

So for the enableDropProtection change, it will simple just return the obj, which should just contains `enableDropProtection` instead of other fields